### ISSUE TITLE
[1.9.4] Profile command shows no results.

### ISSUE THREAD

**Issue by LuxusDarkangel** [MinimallyCorrect/TickProfiler#50]
Log: https://gist.github.com/LuxusDarkangel/0273e8faf7b2a89811533ad6d7c20563

Happens in pure Forge and using SpongeForge.

**Comment by LunNova** [comment]
Working fine here. About 90% sure it's because you tested with no users online and waited a while before testing.

Entities stop getting ticked after 1200 ticks if no-one's online and there are no forced chunks.

**Comment by LuxusDarkangel** [comment]
In this case you're totally right, but the issue is still there. I tested it first in-game using SpongeForge, and then proceeded to test it in a pure Forge server using only the console, as shown in the log, to make sure it wasn't a Sponge issue. It is, If you install latest SpongeForge version for 1.9.4, it doesn't display profiling results even while being in-game and making sure entities are getting ticked.

**Comment by LunNova** [comment]
So it's working fine with just forge, only an issue with sponge? I made a silly commit just then which probably doesn't actually fix it. Looks like some sort of mappings issue:

With Sponge:

```
[23:33:13] [main/TRACE] [JavaPatcher/]: Patching net.minecraft.world.WorldServer with clone(deobf:net.minecraft.world.WorldServer/updateEntities,to:updateEntities_withProfiling) {func_72939_s()V}
[23:33:13] [main/TRACE] [JavaPatcher/]: Patching net.minecraft.world.WorldServer with insertCodeBefore(deobf:net.minecraft.world.WorldServer/updateEntities) {func_72939_s()V}
[23:33:13] [main/TRACE] [JavaPatcher/]: Patching net.minecraft.world.WorldServer with replaceMethodCall(deobf:net.minecraft.world.WorldServer/updateEntities_withProfiling,method:func_73660_a) {updateEntities_withProfiling()}
[23:33:13] [main/WARN] [JavaPatcher/]: Didn't find any method calls to replace in net.minecraft.world.WorldServer.updateEntities_withProfiling(). Class: null, method: func_73660_a, index: -1
[23:33:13] [main/TRACE] [JavaPatcher/]: Patching net.minecraft.world.WorldServer with replaceMethodCall(deobf:net.minecraft.world.WorldServer/updateEntities_withProfiling,method:func_72870_g) {updateEntities_withProfiling()}
[23:33:13] [main/WARN] [JavaPatcher/]: Didn't find any method calls to replace in net.minecraft.world.WorldServer.updateEntities_withProfiling(). Class: null, method: func_72870_g, index: -1
```

Going to sleep now, need to get up early tomorrow so staying up to fix this is a bad idea.

**Comment by LuxusDarkangel** [comment]
Yeah, it's not fixed yet. We'll see tomorrow then.

**Comment by LunNova** [comment]
Sponge patching of updateEntities has changed to https://github.com/SpongePowered/SpongeCommon/blob/1.9.4/src/main/java/org/spongepowered/common/mixin/core/world/MixinWorld.java#L812

May be able to fix this by **removing** the sponge-specific patch. Unexpected, but nice.

**Comment by LunNova** [comment]
Can you confirm if that fix worked in 1.9.4 build 4?

edit: Marking as 1.10 as this change will also apply to future sponge releases on 1.10.

**Comment by LuxusDarkangel** [comment]
Yes, it worked.

Guess the same issue will happen with 1.10, but I don't have time to test it right now, will do later.

**Comment by LunNova** [comment]
Thanks, fix also in 1.10 branch now.
